lib/SimpleSAML/HTTP/RunnableResponse.php

```diff
@@ -31,6 +31,7 @@ public function __construct(callable $callable, $args = [])
     {
         $this->arguments = $args;
         $this->callable = $callable;
+        $this->charset = 'UTF-8';
```

Comment:
This is an upstream-issue.. Symfony is not initializing this var, even on their latest version